Build/GameSettings.h

```diff
@@ -57,6 +58,7 @@ enum
 struct GAME_SETTINGS
 {
 	INT8				bLastSavedGameSlot;							// The last saved game number goes in here
+	wchar_t     sCurrentSavedGameName[SIZE_OF_SAVE_GAME_DESC];                     // The name of the current Savegame goes here. Only relevant for Dead is Dead games
```

Comment:
Can we use an UTF8 string for that instead? I would rather not introduce new wchar_t strings.